libs/gl-client/Cargo.toml

```diff
@@ -10,21 +10,28 @@ export = ["chacha20poly1305", "secp256k1"]
 
 [dependencies]
 anyhow = "*"
+arti-client = {version="0.9.2", features=["onion-service-client"]}
 base64 = "^0.21"
+bech32 = "0.9.1"
 bytes = "1.2.1"
 hex = "*"
 http = "0.2"
 http-body = "^0.4"
+lightning-invoice = "0.23.0"
 log = "^0.4"
 pin-project = "*"
 prost = "0.11"
+reqwest = {version="0.11.18", features=["blocking", "json"]}
```

Comment:
Everything we do is `async`, why are you using `blocking` here? It will block the async reactor thread arbitrarily while the request is in flight.